data/gegno/gegno I corroboration.txt

```diff
@@ -0,0 +1,1226 @@
+				goto imagine
+
+			label imagine
+			`	"Imagine, if you will, during the substantial conflicts the humans call the Turf Wars, a strange alien came to human space. We Quarg woefully witnessed that hazardous conflict - it was a time of great waste, unfortunate peril, and unsavory distrust among humanity. The Gegno are in a vulnerable state notably comparable to such a time, and introduction of outsiders would not befit the case. The truth remains that we have been contacted by a few Gegno of importance in respect to your past presences, and there has been show of concern to your unstated intentions." After a brief pause, the Quarg looks to you in a strange manner, and you aren't sure what to make of it. "Though, unexpectedly, a select few times we have also been curiously asked of your origins, an experience we are not strongly familiar with involving the Gegno."`
+			`	The Quarg leaves its chair and motions you to do the same.`
+			choice
+				`	(Stand up.)`
+
+			`	"Our best of recommendations to you is to avoid further contact in the coming future. Meanwhile, we shall attempt a most careful negotiation with their hierarchy to come to terms of a ground of acceptance for your presence. Should we find commonality, you will receive communication from one of the title 'Adrauni', as has been bestowed upon me."`
+			`	The rest of the Quarg, who have been listening to the conversation attentively the whole time, begin to escort you out of the room. If the Gegno are willing to go as far as to contact the Quarg about your appearances, then this is truly an important matter to them. For the time being, it's probably best to allow the Quarg to find some kind of resolution with the Gegno while you keep your distance.`
+				decline
+
+
+			label "negative rep"
+			`	"Formerly, I have offered suggestions of heedful interplay regarding the Gegno. However, we are increasingly aware of correlative happenings upon them involving yourself. It would seem consequential events have indeed occurred. Not often previously were we approached by Gegno, but on a few notable occasions now have meaningful figures sought us out for inquiry - most notably due to violent circumstances." The Quarg stares at you for an extended period of time, waiting for some kind of answer.`
+			choice
+				`	"They attacked me first."`
+				`	"I wanted to get their attention."`
+					goto attention
+				`	"Violence happens all the time in human space, but the Quarg don't seem interested in it."`
+					goto violence
+
+			`	The Quarg once again pauses for a long moment, staring at you. "Based on some principles, I may see reason for them to be, inclined, to do such. However, to do so in this case would not line with their elder's interest, nor would any, fabrications." The Quarg blinks at you, then ponders for a moment.`
```

Comment:
It feels weird to start this off with "once again pauses" when it is after our saying something. Before the choice, the Quarg was already just staring at you for a while, so saying it paused again doesn't seem right as it wasn't doing anything. Maybe change it to be stating like in the others how it takes a bit to consider what you said or blink or whatever.